shopinvader_api_address/tests/test_shopinvader_address_api.py

```diff
@@ -49,7 +49,7 @@ def setUpClass(cls) -> None:
         cls.default_fastapi_authenticated_partner = cls.test_partner
         cls.default_fastapi_router = address_router
 
-    def test_get_invoicing_address(self):
+    def test_get_billing_address(self):
```

Comment:
Why renaming this test from invoicing to billing? Because it does a get of /addresses/invoicing.

Reply:
I reverted it